### PR TITLE
Grading endpoint now also shows student's group

### PR DESCRIPTION
**Pull request by rrtheonlyone**
Fixes Issue #262 

This is to help admin staff keep track of how students in various groups are performing.

### CODE REVIEW

**Comment by indocomsoft** [comment]
Please use github keyword "fixes" instead of "addresses issue" as using the former will auto-close the issue

File: lib/cadet/assessments/assessments.ex [hunk 1]
```diff
@@ -442,20 +442,22 @@ defmodule Cadet.Assessments do
         s.id == x.submission_id
       )
       |> join(:inner, [s], st in assoc(s, :student))
+      |> join(:inner, [s, x, st], g in assoc(st, :group))
```
Comment (indocomsoft) [inline lib/cadet/assessments/assessments.ex]: `[_, _, st]`

File: lib/cadet/assessments/submission.ex [hunk 2]
```diff
@@ -11,6 +11,7 @@ defmodule Cadet.Assessments.Submission do
     field(:xp, :integer, virtual: true)
     field(:xp_adjustment, :integer, virtual: true)
     field(:xp_bonus, :integer, default: 0)
+    field(:group, :string, virtual: true)
```
Comment (indocomsoft) [inline lib/cadet/assessments/submission.ex]: `:group_name` is a better name.

File: lib/cadet_web/controllers/grading_controller.ex [hunk 3]
```diff
@@ -155,6 +155,7 @@ defmodule CadetWeb.GradingController do
             xp(:integer, "xp earned")
             xpAdjustment(:integer, "xp adjustment given")
             adjustment(:integer, "grade adjustment given")
+            group(:string, "name of student's group")
```
Comment (indocomsoft) [inline lib/cadet_web/controllers/grading_controller.ex]: similarly, `groupName`

**Comment by rrtheonlyone** [comment]
@indocomsoft have done the necessary style changes!

**Comment by indocomsoft** [comment]
LGTM.

@rrtheonlyone btw do you have merge permission?

**Comment by indocomsoft** [comment]
Also, change your "addresses issue" to "fixes" due to reason mentioned earlier

**Comment by rrtheonlyone** [comment]
Done!

Yup I think I have merge permission @indocomsoft

**Pull request merged**